docs/UserGuide.md

```diff
@@ -290,166 +306,153 @@ The `-c` flag indicates the property's new characteristics.<br>
 The `-o` flag indicates the property's owner's new name.<br>
 The `-ph` flag indicates the property's owner's new phone number.<br>
 
-Note that we are updating existing entries in the database, hence all parameters above are optional,
-as we can choose to update a parameter without changing the others. Also, the `INDEX` field specifies the index of the property that we want to update
-on the current visible property list, so if the property is not on this list, then we cannot execute this command on the property.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Only the characteristics fields can be edited to "Not Specified" by entering an empty flag.
+e.g. `-c  `.
+</div>
+
 
 Examples:<br>
 `editprop 3 -n Hill Residence -a Block 225 -ph 750000`: Edits property at index 3 of the list to have a new name Hill Residence, a new address Block 225 and price 750000.
 
 ### List Commands
 #### List buyers in database: `listbuyers`
 
-Lists all buyers in the database in the visible list, that is, removes all filters.
-
-This command updates the buyers list to display all current entries.
-
+Lists all buyers in the database in the visible list, that is, removes all filters.<br>
```

Comment:
Actually I added the second extra line `This command updates...` because I think it was Richard? or Zsigmond that mentioned the first line might not make sense on its own

Reply:
Oh okay I see where that's coming from, will change!